src/main/java/org/telegram/bot/engine/ImageSource.java

```diff
@@ -0,0 +1,28 @@
+package org.telegram.bot.engine;
+
+import java.io.File;
+import java.util.List;
+
```

Comment:
How do you solve the problem of instance inconsistency?
You load files only on setup:
1. Add files to google drive
2. Run app - it load all images from drive
3. Add some more files - it won't load it.

Polling using Scheduler could help: every hour load bunch of images from the drive.